src/bridge/test-packages.c

```diff
@@ -98,6 +97,9 @@ setup (TestCase *tc,
   if (fixture->expect)
     cockpit_expect_warning (fixture->expect);
 
+  if (fixture->lang)
+      g_setenv ("LANG", fixture->lang, TRUE);
```

Comment:
It's common for environments to set LC_* variables (e. g. because you live in Sweden and want dates, money, numbers, etc. formatted in Swedish, but show messages in English), so for robustness this should set `LC_ALL`.

Reply:
Set else where.